.github/workflows/test.yaml

```diff
@@ -171,103 +54,218 @@ jobs:
                 machineFlavor: ${{ secrets.NODE_GROUP_FLAVOR }}
                 machineCount: 2
 
-      #####
-      # For the smoke test, we do a clean deployment of the latest supported version
-      #####
+      - name: Apply network configuration
+        run: echo "$NETWORKING" >> ./values.yaml
+        env:
+          NETWORKING: |
+            clusterNetworking:
+              internalNetwork:
+                networkFilter:
+                  tags: capi-helm-chart-ci
+        if: ${{ !inputs.tests-full }}
```

Comment:
why do we not want to create a new network if we're doing a quick test?

Reply:
Because the most common failure is quota error on routers :)